misk-hibernate/src/main/kotlin/misk/hibernate/ReflectionQueryFactory.kt

```diff
@@ -758,9 +758,8 @@ internal class ReflectionQuery<T : DbEntity<T>>(
     }
   }
 
-  internal fun addConstraint(predicateFactory: PredicateFactory): ReflectionQuery<T> {
-    constraints.add(predicateFactory)
-    return this
+  override fun addConstraint(block: PredicateFactory) {
```

Comment:
I think this is the culprit for the failing Unit tests?

Reply:
Yep. Needed to return the right result. Fixed!